pkgs/by-name/ga/gama/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  expat,
+  texinfo,
+  zip,
+  octave,
+  libxml2,
+
+  # passthru.tests.run
+  runCommand,
+}:
+
+stdenv.mkDerivation (finalAttrs:  {
+  pname = "gama";
+  version = "2.26";
+
+  src = fetchurl {
+    url = with finalAttrs; "mirror://gnu/${pname}/${pname}-${version}.tar.gz";
+    hash = "sha256-8zKPPpbp66tD2zMmcv2H5xeCSdDhUk0uYPhqwpGqx9Y=";
+  };
+
+  buildInputs = [expat];
+
+  nativeBuildInputs = [texinfo zip];
+
+  nativeCheckInputs = [octave libxml2];
+  doCheck = true;
+
+  passthru.tests.run = runCommand "gama-test-run" {
+    local = ./.;
+    nativeBuildInputs = [finalAttrs.finalPackage];
+  } ''
+    diff -U3 --color=auto <(gama-local $local/example.xml) <(cat $local/result.xml);
+    diff -U3 --color=auto <(gama-local $local/example.yaml) <(cat $local/result.yaml);
```

Comment:
The test xml files contain system specific info so the CI fails for non `x86_64-linux`

Maybe only test the yaml since that seems reproducible, that should be sufficient.

Reply:
Oki, did that let's see if it passes now